### ISSUE TITLE
"error CS9027: Unexpected keyword 'unchecked'" is reported for cref syntax

### ISSUE THREAD

**Issue by AlekseyTs** [dotnet/roslyn#69142]
```
        [Fact, WorkItem(60394, "https://github.com/dotnet/roslyn/issues/60394")]
        public void UnqualifiedOperatorMember1_Unchecked()
        {
            UsingNode("operator unchecked +", TestOptions.RegularWithDocumentationComments,
                // (1,16): warning CS1584: XML comment has syntactically incorrect cref attribute 'operator unchecked +'
                // /// <see cref="operator unchecked +"/>
                Diagnostic(ErrorCode.WRN_BadXMLRefSyntax, "operator unchecked +").WithArguments("operator unchecked +").WithLocation(1, 16),
                // (1,25): error CS9027: Unexpected keyword 'unchecked'
                // /// <see cref="operator unchecked +"/>
                Diagnostic(ErrorCode.ERR_MisplacedUnchecked, "unchecked").WithLocation(1, 25));

            N(SyntaxKind.OperatorMemberCref);
            {
                N(SyntaxKind.OperatorKeyword);
                N(SyntaxKind.PlusToken);
            }
            EOF();
        }
```

Expected: A warning. I think we never report errors for doc comments.

Introduced in https://github.com/dotnet/roslyn/pull/61309.
